policyengine_us/variables/gov/states/ne/tax/income/adjusted_gross_income/subtractions/ne_military_retirement_substraction.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class ne_military_retirement_subtraction(Variable):
```

Comment:
looking at the [law](https://nebraskalegislature.gov/laws/statutes.php?statute=77-2716):
I think the person has to be at least 67

Could you add this condition?

![image](https://github.com/user-attachments/assets/39b04619-7c80-4a2a-b90c-df7b918c2e88)